src/components/Header/Tools.tsx

```diff
@@ -27,6 +27,7 @@ import { ScalprumComponent, ScalprumComponentProps } from '@scalprum/react-core'
 import { drawerPanelContentAtom } from '../../state/atoms/drawerPanelContentAtom';
 import { Label } from '@patternfly/react-core/dist/dynamic/components/Label';
 import UsersIcon from '@patternfly/react-icons/dist/dynamic/icons/users-icon';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
```

Comment:
You should not call useChrome in Chrome it's for external modules. The chrome API is available on an internal Chrome context (InternalChromeContext). Getting from the context is preferable in Chrome.